packages/ui/src/components/va-sidebar/VaSidebar.vue

```diff
@@ -7,7 +7,7 @@
     @mouseenter="updateHoverState(true)"
     @mouseleave="updateHoverState(false)"
   >
-    <div class="va-sidebar__menu" :style="`width: ${computedWidth};`">
```

Comment:
Hoverable minimized story seems to be broken (Should do somethin on hover? right now - does nothing)

<img width="407" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/dd0374a8-f33c-4bab-839b-fa67daae2a56">